prow/Makefile

```diff
@@ -53,7 +53,7 @@ CLUSTER       ?= prow
 
 # Build and push specific variables.
 REGISTRY ?= gcr.io
-PUSH     ?= gcloud docker -- push
+PUSH     ?= docker push
```

Comment:
eh, now we assume you have gcr.io credentials by default?

I guess this is fine now, as `gcloud docker -- push` spills out error now if I have `docker-credential-gcr` set up.. If so we probably should update this across our repo.

Reply:
How did I set that up? And/or how do I make gcloud docker work again?

Reply:
what error do you get if you use gcloud docker?

Reply:
```
gcloud docker -- pull gcr.io/k8s-testimages/hook:latest
ERROR: Docker CLI operation failed:

Error saving credentials: error storing credentials - err: exit status 1, out: `docker-credential-gcr/helper: this operation is unsupported for GCR, please see docker-credential-gcr documentation for supported login methods. 'gcloud docker' is unnecessary when using docker-credential-gcr, use 'docker' instead.`

ERROR: (gcloud.docker) Docker login failed.
```

Reply:
I'm thinking it is a new version of gcloud that did this?

Reply:
yeah, that's https://github.com/GoogleCloudPlatform/docker-credential-gcr/issues/21

I think they fixed the `gcloud docker` hack in latest gcloud, we probably can update using docker push everywhere for our images :-)